Softeq.XToolkit.PushNotifications/PushTokenSynchronizer.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Developed for PAWS-HALO by Softeq Development Corporation
+// http://www.softeq.com
+
+using System;
+using System.Threading.Tasks;
+using Softeq.XToolkit.PushNotifications.Abstract;
+
+namespace Softeq.XToolkit.PushNotifications
+{
+    public class PushTokenSynchronizer : IPushTokenSynchronizer
```

Comment:
I would be better, if `PushTokenSynchronizer` would handle everything connected to communication with backend, including "unregistration"

Reply:
Fixed